L1Trigger/L1TTrackMatch/python/L1TkHTMissProducer_cfi.py

```diff
@@ -17,12 +17,12 @@
      displaced = cms.bool(False) #Run with prompt/displaced jets - only useful for track jets
 )
 
-L1TkCaloHTMissVtx = L1TkCaloHTMiss.clone()
-L1TkCaloHTMiss.doVtxConstrain = cms.bool(True)
+l1tTkCaloHTMissVtx = l1tTkCaloHTMiss.clone()
+l1tTkCaloHTMiss.doVtxConstrain = cms.bool(True)
```

Comment:
You are modifying the module from which youo are cloning, not the cloned one. I think you wanted to do the opposite.
A better (less error prone) alternative to modify the parameters in cloned modules is:
```suggestion
l1tTkCaloHTMissVtx = l1tTkCaloHTMiss.clone(doVtxConstrain = True)
```
(if this is what you actually wanted to do)

Reply:
Done